CSS/Indentation Rules.tmPreferences

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+	<key>scope</key>
+	<string>source.css</string>
+	<key>settings</key>
+	<dict>
+		<key>decreaseIndentPattern</key>
+		<string>^\s*[)\]}]</string>
+		<key>increaseIndentPattern</key>
+		<string><![CDATA[(?x)
+			# any text
+			^.*
+			# follwed by opening bracket
+			[(\[{]
+			# optionally followed by...
+			(?<balanced>
+			# balanced parenthesis
+			  \( (?: \g<balanced> | [^)] )* \)
+			# balanced brackets
+		  	| \[ (?: \g<balanced> | [^]] )* \]
+			# balanced braces
+			| \{ (?: \g<balanced> | [^}] )* \}
+			# double quoted string with ignored escaped quotation marks
+			| \" .*? (?<![^\\]\\)(?<![\\]{3})\"
+			# single quoted string with ignored escaped quotation marks
+			| \' .*? (?<![^\\]\\)\'
+			# block comment
+			| /\* .*? \*/
+			# anything but closing brackets
+			| [^])}]
+			)*
+			# ... until end of line
+			$
+		]]></string>
+		<key>unIndentedLinePattern</key>
+		<string>^\s*(?:(?:/\*|\*/).*)?$</string>
+		<key>preserveIndent</key>
+		<true/>
```

Comment:
While `preserveIndent` helps with multi-line block strings I have a feeling it breaks detenting text somehow.

```
a {
    color: red;
    }| <- not detended when entered manually, if `preserveIdent = true`
```

Reply:
The following commit should fix this issue.